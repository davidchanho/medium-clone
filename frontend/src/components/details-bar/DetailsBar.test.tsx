import React from "react";
import renderer from "react-test-renderer";
import DetailsBar from ".";

describe("DetailsBar", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<DetailsBar />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
