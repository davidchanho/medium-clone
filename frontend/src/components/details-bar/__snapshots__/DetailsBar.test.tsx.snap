// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailsBar renders correctly 1`] = `
<div
  className="w-25 card"
>
  <h2>
    <div
      className="bg-white mb-3 card-header"
    >
      <p
        className="mb-3"
      >
        
      </p>
      <button
        className="btn btn-primary"
        disabled={false}
        type="button"
      >
        Follow
      </button>
    </div>
     
    <div
      className="dropdown-divider"
      role="separator"
    />
    <div
      className="card-body list-group"
      onKeyDown={[Function]}
    >
      <div
        className="d-flex align-items-center list-group-item"
        data-rb-event-key={null}
        disabled={false}
        onClick={[Function]}
      >
        <svg
          className="mr-3"
          fill="currentColor"
          height="1em"
          onClick={[Function]}
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": undefined,
            }
          }
          viewBox="0 0 448 512"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M372.57 112.641v-10.825c0-43.612-40.52-76.691-83.039-65.546-25.629-49.5-94.09-47.45-117.982.747C130.269 26.456 89.144 57.945 89.144 102v126.13c-19.953-7.427-43.308-5.068-62.083 8.871-29.355 21.796-35.794 63.333-14.55 93.153L132.48 498.569a32 32 0 0 0 26.062 13.432h222.897c14.904 0 27.835-10.289 31.182-24.813l30.184-130.958A203.637 203.637 0 0 0 448 310.564V179c0-40.62-35.523-71.992-75.43-66.359zm27.427 197.922c0 11.731-1.334 23.469-3.965 34.886L368.707 464h-201.92L51.591 302.303c-14.439-20.27 15.023-42.776 29.394-22.605l27.128 38.079c8.995 12.626 29.031 6.287 29.031-9.283V102c0-25.645 36.571-24.81 36.571.691V256c0 8.837 7.163 16 16 16h6.856c8.837 0 16-7.163 16-16V67c0-25.663 36.571-24.81 36.571.691V256c0 8.837 7.163 16 16 16h6.856c8.837 0 16-7.163 16-16V101.125c0-25.672 36.57-24.81 36.57.691V256c0 8.837 7.163 16 16 16h6.857c8.837 0 16-7.163 16-16v-76.309c0-26.242 36.57-25.64 36.57-.691v131.563z"
          />
        </svg>
        0
      </div>
      <div
        className="d-flex align-items-center justify-content-between list-group-item"
        data-rb-event-key={null}
        disabled={false}
        onClick={[Function]}
      >
        <svg
          className="mr-3"
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": undefined,
            }
          }
          viewBox="0 0 24 24"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M20,2H4C2.897,2,2,2.897,2,4v12c0,1.103,0.897,2,2,2h3v3.767L13.277,18H20c1.103,0,2-0.897,2-2V4C22,2.897,21.103,2,20,2z M20,16h-7.277L9,18.233V16H4V4h16V16z"
          />
        </svg>
         
        0
      </div>
      <span
        onClick={[Function]}
      >
        <svg
          fill="currentColor"
          height="22"
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": undefined,
            }
          }
          viewBox="0 0 16 16"
          width="22"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            clipRule="evenodd"
            d="M8 12l5 3V3a2 2 0 00-2-2H5a2 2 0 00-2 2v12l5-3zm-4 1.234l4-2.4 4 2.4V3a1 1 0 00-1-1H5a1 1 0 00-1 1v10.234z"
            fillRule="evenodd"
          />
        </svg>
      </span>
    </div>
  </h2>
</div>
`;
